src/study.js

```diff
@@ -88,10 +88,10 @@ export default async function runStudy(devMode) {
         plugins: [
```

Comment:
Please also change the Glean application id on line 86 to `"rally-study-zero-one"` instead of `"rally-zero-one"` (only for Glean).